src/app/modules/consultations/consultations-summary/consultations-summary.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="summary-index">
+  <div class="consultation-profile">
+      <section class="consultation-profile__sidebar-section">
+        <app-profile-card [profile]='profileData' [summaryData]="summaryData"></app-profile-card>
+      </section>
+      <section class="consultation-profile__main-section">
+
+        <div *ngIf="publicResponses?.length">
+          <div class="d-flex justify-content-between align-items-center">
+            <p class="text-18 text-16-sm m-0 bold-text">Public responses</p>
+            <a href="javascript:void(0)" class="what-resp-text">What are public responses?</a>
+          </div>
+
+          <div class="paper-effect my-4 m-0-sm ">
+            <ng-container *ngFor="let response of publicResponses">
+              <div class="response-container">
+                <div class="d-inline-flex align-items-center">
+                  <img class="avatar avatar--42" src="https://i0.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?resize=256%2C256&quality=100&ssl=1" />
+                  <p class="regular-text bold-text m-0 ml-2">{{response?.node?.user?.firstName}}</p>
+                </div>
+                <p class="text-16">{{response.node?.responseText}}</p>
+                <div class="vote-section">
+                  <div class="vote-count">
+                    <div [ngClass]="response?.node.votedAs?.voteDirection === 'up'  ? 'active' : ''">
+                        <span (click)="vote('up', response?.node)" [ngClass]="response?.node.votedAs?.voteDirection === 'up' ? 'active__btn' : ''"><i class="fa fa-thumbs-up"></i></span>
```

Comment:
@Nishanthh No need for this in the summary page. Summary page should have read only information